app/presenters/nsm/incomplete_items.rb

```diff
@@ -0,0 +1,78 @@
+module Nsm
+  class IncompleteItems
+    include ActionView::Helpers::TagHelper
+    include ActionView::Helpers::TranslationHelper
+
+    # govuk_link_to required modules
+    include GovukLinkHelper
+    include GovukVisuallyHiddenHelper
+    include ActionView::Helpers::UrlHelper
+    # for path methods
+    include Routing
+
+    attr_reader :controller
+
+    EXPECTED_ITEM_TYPES = [:work_items, :disbursements].freeze
+
+    def initialize(claim, type, controller)
+      raise "Invalid item type: #{type}" unless type.in?(EXPECTED_ITEM_TYPES)
+
+      @claim = claim
+      @type = type
+      @controller = controller
+    end
+
+    def incomplete_items
+      items.reject(&:complete?).sort_by(&:position)
+    end
+
+    def summary
+      "#{t("#{path_key}.incomplete_summary", count: incomplete_items.count)}: #{links}"
+    end
+
+    def links
+      incomplete_items.map do |item|
+        govuk_link_to(
+          path_title(item),
+          path_url(item)
+        )
+      end.join(', ')
+    end
+
+    private
+
+    def items
+      if @type == :work_items
+        @items ||= @claim.work_items
+      elsif @type == :disbursements
+        @items ||= @claim.disbursements
+      # :nocov:
+      else
+        #  no test coverage needed as inaccessible  due to initializer
+        false
+      end
+      # :nocov:
+    end
+
+    def path_title(item)
+      t("#{path_key}.item", index: item.position)
+    end
+
+    def path_url(item)
+      if item.is_a? WorkItem
```

Comment:
https://api.rubyonrails.org/classes/ActionDispatch/Routing/PolymorphicRoutes.html

Reply:
forgot to report back but not 100% sure polymorphic paths are possible with the namespacing situation we have - or at least it's not too clear from the documentation, playing/reading around now though